output/tlf-demographic.out.pilot3

```diff
@@ -0,0 +1,40 @@
+Age                                                                                              
+  Mean (sd)                           75.21 (8.59)        75.67 (8.29)           74.38 (7.89)    
+  Median                                  76.00               77.50                 76.00        
+  Min - Max                           52.00 - 89.00       51.00 - 88.00         56.00 - 88.00    
+Pooled Age Group 1                                                                               
+  <65                                      14                   8                     11         
+  65-80                                    42                  47                     55         
+  >80                                      30                  29                     18         
+Race                                                                                             
+  WHITE                                    78                  78                     74         
+  BLACK OR AFRICAN AMERICAN                 8                   6                     9          
+  AMERICAN INDIAN OR ALASKA NATIVE          0                   0                     1          
+Baseline Height (cm)                                                                             
+  Mean (sd)                          162.57 (11.52)      163.43 (10.42)         165.82 (10.13)   
+  Median                                 162.60              162.60                 165.10       
+  Min - Max                          137.20 - 185.40     135.90 - 195.60       146.10 - 190.50   
+Baseline Weight (kg)                                                                             
+  Mean (sd)                           62.76 (12.77)       67.27 (14.12)         70.00 (14.66)    
+  Median                                  60.55               64.90                 69.20        
+  Min - Max                           34.00 - 86.20      45.40 - 106.10         41.70 - 108.00   
+Baseline BMI (kg/m^2)                                                                            
+  Mean (sd)                           23.63 (3.67)        25.06 (4.27)           25.35 (4.16)    
+  Median                                  23.40               24.30                 24.80        
+  Min - Max                           15.00 - 33.30       17.70 - 40.10         13.70 - 34.60    
+MMSE Total                                                                                       
+  Mean (sd)                           18.05 (4.27)        17.87 (4.22)           18.51 (4.16)    
+  Median                                  19.50               18.00                 20.00        
+  Min - Max                           10.00 - 23.00       10.00 - 24.00         10.00 - 24.00    
+—————————————————————————————————————————————————————————————————————————————————————————————————
+
+Program: tlf_demographic.Rmd 
+2023-02-24 11:35:44
+
```

Comment:
For the filename of the output, can you please update to `tlf-demographic-pilot3.out`?

Reply:
this is fixed.  Thanks, @wangben718 !